src/Lib.hs

```diff
@@ -154,42 +155,22 @@ fetchTickets = do
     putTextLn "All the tickets has been processed."
 
 
-showStatistics :: App ()
-showStatistics = do
-    cfg <- ask
-
-    let email   = cfgEmail cfg
-    let userId  = UserId . fromIntegral $ cfgAgentId cfg
-
-    -- We first fetch the function from the configuration
-    listTickets <- asksZendeskLayer zlListAssignedTickets
-
-    putTextLn $ "Classifier is going to gather ticket information assigned to: " <> email
-
-    tickets     <- listTickets userId
-    pure () -- TODO(ks): Implement anew.
-
-
 -- TODO(ks): Extract repeating code, generalize.
 listAndSortTickets :: App [TicketInfo]
 listAndSortTickets = do
-
     Config{..}  <- ask
 
-    listAgents <- asksZendeskLayer zlListAdminAgents
-    agents <- listAgents
+    let email   = cfgEmail
```

Comment:
We want to use `listAgents` since we want to analyze all the tickets not just tickets from a single agent. Seems like you merged this incorrectly.

Reply:
Should be fixed now